### PR TITLE
feat: basic home feature added

### PR DESCRIPTION
**Pull request by JowelTisso**
This PR is for Home screen :
* Some navigation feature has been added
* And minor changes in other files.

### CODE REVIEW

File: src/pages/products/Products.jsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 import ProductCard from "../../components/productCard/ProductCard";
 import "./Products.css";
 
-const Products = () => {
+const Products = (props) => {
   const [sliderValue, setSliderValue] = useState(0);
   const [products, setProducts] = useState([]);
 
+  const data = useLocation();
+  console.log(data);
```
Comment (rahulmistry751) [inline src/pages/products/Products.jsx:13]: remove console.log

**Review by rahulmistry751 — APPROVED** [review]
Clean code

File: src/pages/products/Products.jsx (hunk 1 above)
Comment (JowelTisso) [reply src/pages/products/Products.jsx:13]: Ok

**Review by shobit1337 — APPROVED** [review]
Your code looks to me. Go ahead and merge.

**Pull request merged**